task/buildah-oci-ta/0.2/buildah-oci-ta.yaml

```diff
@@ -348,10 +348,18 @@ spec:
           BUILD_ARG_FLAGS+=("--build-arg=$build_arg")
         done
 
-        BASE_IMAGES=$(
-          dockerfile-json "${BUILD_ARG_FLAGS[@]}" "$dockerfile_copy" |
-            jq -r '.Stages[] | select(.From | .Stage or .Scratch | not) | .BaseName | select(test("^oci-archive:") | not)'
-        )
+        if ! dockerfile_json_output=$(
```

Comment:
How about using

```
set -euo pipefail
```

and have it safe for everyhthing in script. I assume that reason why it doesn't fail is missing `-o pipefail` and thus error is hidded.


https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425